src/applications/vaos/services/appointment/transformers.js

```diff
@@ -125,17 +128,22 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruthTelehealth) {
 
   if (isRequest) {
     const { requestedPeriods, created } = appt;
-    const reqPeriods = requestedPeriods?.map(d => ({
-      // by passing the format into the moment constructor, we are
-      // preventing the local time zone conversion from occuring
-      // which was causing incorrect dates to be displayed
-      start: `${moment(d.start, 'YYYY-MM-DDTHH:mm:ss').format(
-        'YYYY-MM-DDTHH:mm:ss',
-      )}.000`,
-      end: `${moment(d.end, 'YYYY-MM-DDTHH:mm:ss').format(
-        'YYYY-MM-DDTHH:mm:ss',
-      )}.999`,
-    }));
+    const reqPeriods = requestedPeriods?.map(d => {
+      const endDate = d.end || d.start;
```

Comment:
From standup, it sounds like we can just remove endDate for requests since we don't use it